tests/buildtest/crates/invalid-cargotoml-missing-patch/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "invalid-cargotoml-missing-patch"
+version = "0.1.0"
+authors = ["Pietro Albini <pietro@pietroalbini.org>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+build-rs = "=0.1.2"
+
+[patch.crates-io]
+build-rs = { git = "https://github.com/rust-lang/rustwide.git", rev = "07784be00b68cfd6bf80006c8d8669a7d6374ec2" }
```

Comment:
can you explain what's missing here?

Reply:
Similar to the `*-missing-override` one, again that commit of this repository doesn't define a package named build-rs so the patch fails.

Reproduction of <https://crater-reports.s3.amazonaws.com/beta-1.86-1/1.85.0/gh/mrvillage.timer/log.txt>